client/src/pages/organizations/Show.tsx

```diff
@@ -219,6 +220,12 @@ const OrganizationShow = ({ pageDispatchers }: OrganizationShowProps) => {
       uuid
     }
   )
+  useEffect(() => {
+    if (data?.organization) {
+      const organization = new Organization(data ? data.organization : {})
+      setAttachments(organization.attachments || [])
+    }
+  }, [data])
```

Comment:
Why not simply:
```javascript
  useEffect(() => {
    setAttachments(data?.organization?.attachments || [])
  }, [data])
```
I would also suggest to put the `useEffect` after `usePageTitle`.